nemo_text_processing/text_normalization/de/taggers/cardinal.py

```diff
@@ -104,7 +104,7 @@ def tens_no_zero():
 
         def hundred_non_zero():
             return (graph_digit_no_one + insert_space | pynini.cross("1", "ein ")) + pynutil.insert("hundert") + (
-                pynini.closure(insert_space + pynutil.insert(AND), 0, 1) + insert_space + tens_no_zero()
+                pynini.closure(insert_space + pynutil.insert(AND, weight=0.0001), 0, 1) + insert_space + tens_no_zero()
```

Comment:
consider adding `0.0001` to de/graph_utils as a default weight constant  since it's been used across many classes

Reply:
I am using english graph_utils